assets/javascripts/discourse/components/event-date.gjs

```diff
@@ -50,11 +49,13 @@ export default class EventDate extends Component {
   }
 
   get eventEndedAt() {
```

Comment:
I wonder if we should give this a better name now?

Reply:
Naming’s hard. I’m not sure what would be better 🤔 Technically, an event with just a start date ends up have his end date == start date. So it’s not _wrong_ to call it “end date”.